x-pack/plugins/security_solution/public/entity_analytics/pages/entity_store_management_page.tsx

```diff
@@ -0,0 +1,456 @@
+              defaultMessage={
+                'This will delete all Security Entity store records. Source data, Entity risk scores, and Asset criticality assignments are unaffected by this action. This operation cannot be undone.'
+              }
+            />
+          </EuiConfirmModal>
+        )}
+      </>
+    );
+  };
+
+  const FileUploadSection: React.FC = () => {
+    if (
+      !hasEntityAnalyticsCapability ||
+      !isAssetCriticalityEnabled ||
+      assetCriticalityPrivilegesError?.body.status_code === 403
+    ) {
+      return <AssetCriticalityIssueCallout />;
+    }
+    if (!hasAssetCriticalityWritePermissions) {
+      return <InsufficientAssetCriticalityPrivilegesCallout />;
+    }
+    return (
+      <EuiFlexItem grow={3}>
+        <EuiTitle size="s">
+          <h2>
+            <FormattedMessage
+              id="xpack.securitySolution.entityAnalytics.assetCriticalityUploadPage.subTitle"
+              defaultMessage="Upload entities using CSV file upload"
+            />
+          </h2>
+        </EuiTitle>
+        <EuiSpacer size="m" />
+        <EuiText size="s">
+          <FormattedMessage
+            id="xpack.securitySolution.entityAnalytics.assetCriticalityUploadPage.description"
+            defaultMessage="Bulk assign asset criticality by importing a CSV, TXT, or TSV file exported from your asset management tools. This ensures data accuracy and reduces manual input errors."
+          />
+        </EuiText>
+        <EuiSpacer size="s" />
+        <AssetCriticalityFileUploader />
+      </EuiFlexItem>
+    );
+  };
+
+  const canDeleteEntityEngine = !['not_installed', 'loading', 'installing'].includes(
+    entityStoreStatus.status
+  );
+
+  return (
+    <>
+      <EuiPageHeader
+        data-test-subj="entityStoreManagementPage"
+        pageTitle={
+          <>
+            <FormattedMessage
+              id="xpack.securitySolution.entityAnalytics.entityStoreManagementPage.title"
+              defaultMessage="Entity Store"
+            />{' '}
+            <EuiToolTip content={TECHNICAL_PREVIEW_TOOLTIP}>
+              <EuiBetaBadge label={TECHNICAL_PREVIEW} />
+            </EuiToolTip>
+          </>
+        }
+        alignItems="center"
+        rightSideItems={
+          !isEntityStoreFeatureFlagDisabled
+            ? [
+                <EnablementButton
+                  isLoading={
+                    initEntityEngineMutation.isLoading ||
+                    stopEntityEngineMutation.isLoading ||
+                    deleteEntityEngineMutation.isLoading ||
+                    entityStoreInstallingStatuses.includes(entityStoreStatus.status)
+                  }
+                  isDisabled={
+                    initEntityEngineMutation.isLoading ||
+                    stopEntityEngineMutation.isLoading ||
+                    deleteEntityEngineMutation.isLoading ||
+                    switchDisabledStatuses.includes(entityStoreStatus.status)
```

Comment:
This set of boolean statements can be abstracted into a single variable.

Reply:
Done